app/views/statuses/_group.html.erb

```diff
@@ -0,0 +1,13 @@
+<div class="status-group">
+  <%= render 'statuses/status', status: group %>
+
+  <div class="status-items">
+    <% group.statuses.each do |status| %>
```

Comment:
This partial is a little bit confusing, for what I could get `group` is a instance of `StatusGroup` and `status` is a instance of `Status` right?

If that is the case I don't see why `StatusGroup` have all those methods.